app/models/architecture.rb

```diff
@@ -16,6 +16,10 @@ class Architecture < ApplicationRecord
 
   scoped_search :on => :name, :complete_value => :true
 
+  class Jail < ::Safemode::Jail
```

Comment:
Is this code related to this PR?

Reply:
It is intentionally not :-)

Reply:
huh? if it's not related please split it to a different PR.

Reply:
Resolved.